{"name": "Проверка ошибки при создании двух одинаковых курьеров", "status": "passed", "description": "Отправляем повторный запрос на создание курьера, проверяем ответ и удаляем курьера", "start": 1723305673237, "stop": 1723305673532, "uuid": "756e6642-da5d-4da2-b88e-75873719dcff", "historyId": "d64c0d867f738829cd30884375940b6d", "testCaseId": "d64c0d867f738829cd30884375940b6d", "fullName": "test_create_courier.TestCreateCourier#test_registration_double_courier_failed", "labels": [{"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "DESKTOP-6NP8JLT"}, {"name": "thread", "value": "73680-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_create_courier"}]}